known_servers.yaml

```diff
@@ -1,5 +1,4 @@
-  - transport01.raiden.network
-  - transport02.raiden.network
-  - transport03.raiden.network
```

Comment:
I think we shouldn't remove our dev servers from the federation whitelist. This would cause a split in the discovery rooms. It'd be enough to just leave them out of the `.main`/`.test` lists fetched by the clients.